demo/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "demo",
+  "private": true,
+  "version": "0.1.0",
+  "dependencies": {
+    "@citizensadvice/design-system": "file:..",
```

Comment:
This loads the design system package from the local project so the demo app always runs against the latest code.